### DISCUSSION TITLE
ability to set "unknown" or "unavailable"

### DISCUSSION THREAD

**Discussion by doctorkb** [zachowj/node-red-contrib-home-assistant-websocket#1395] [Feature Request]
As a user, I'd like the ability to set an "unknown" or "unavailable" (or "none") state to a Node-RED-created entity so that I can mark sensors as unused.

Real-life situation: I have a grill thermometer that can send six temperature points. I often use only two. I would like to send the rest as a non-numerical value (null, unknown, unavailable, none, etc.) so as to not create strange data points on the Home Assistant graphs.

**Comment by zachowj** [comment]
duplicate #1085
